pkg/entity/entity.go

```diff
@@ -57,19 +67,38 @@ func (es Entities) String() string {
 func (es Entities) ColorString() string {
 	var line strings.Builder
 	for _, e := range es {
-		s := e.Text
 		switch e.Type {
 		case TypeIPv4:
-			s = color.GreenString(e.Text)
+			line.WriteString(color.GreenString(e.Text))
 		case TypeIPv6:
-			s = color.BlueString(e.Text)
+			line.WriteString(color.BlueString(e.Text))
 		case TypeDomain:
-			s = color.YellowString(e.Text)
+			line.WriteString(color.YellowString(e.Text))
+		default:
+			line.WriteString(e.Text)
 		}
-		if e.Type != TypePlain && len(e.Info) > 0 {
-			s += " [" + color.RedString(e.Info) + "] "
+		if e.Type != TypePlain {
+			if len(e.InfoText) > 0 {
+				line.WriteString(" [" + color.RedString(e.InfoText) + "] ")
+			}
 		}
-		line.WriteString(s)
+		line.WriteString("\n")
+	}
+	return strings.TrimSpace(line.String())
```

Comment:
It wasn't just a change in code style. because the original `nali 1.1.1.1 2.2.2.2` output to a one line . I fixed this bug. Of course I will separate this pr later.